findmin.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import("fmt")
+
+func findmin(in []int) int{
+	smallest := in[0]
+    for i := 0; i < len(in); i ++ {
+        var x = in[i]
```

Comment:
Hi. You can start from in[1], because in[0] is already in smallest variable. You do not need to compare in[0] and in[0].